assets/css/style.css

```diff
@@ -184,11 +196,87 @@ blockquote {
     height: 80vh !important;
 }
 
-.osicon {
+.osicon, .obsicon {
     padding-right: 1ex;
     margin-bottom: 1ex;
 }
 
+.obsicon {
+    float: left;
+}
+
+.dlc .osimg {
+}
+
+.dlc input {
+    opacity: 0;
+    position: fixed;
+}
+.dlc .info h3 {
+    display: none;
+}
+
+.dlc .icon {
+    display: inline-block;
+    width: 105px;
```

Comment:
yea. don't want to make it wider (no good on small screens), fixed it with /RHEL now